CHANGELOG.md

```diff
@@ -16,13 +20,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
```

Comment:
Something's gone awry here - we've got two `## [Unreleased]` sections. Might be worth holding a sec to fix until I merge #1236 and maybe merge some of the other PRs so the changelog clashes get smoothed out.

Reply:
I did that intentionally when I added the original changelog entry. This conflict didn't impact the changelog.

Reply:
Ahhh, I see what happened. I'll fix.

Reply:
Ok, better now.